docs/developer-guide/release.md

```diff
@@ -105,19 +102,19 @@ Press Enter to open github.com in your browser...
     3. Ensure it has all release artifacts attached to it (there should be 22 assets in the draft. Note, there will be 24 after the release is published.)
     4. Check this file to ensure you see the expected changes (spot check a few changes): download the mondo.obo or mondo.owl from the asset list in the release.
 1. Write a description of the release
-   1. Add the release description to the release tab:
-       - All of the releases can be found under the [releases](https://github.com/monarch-initiative/mondo/releases) tab.
-       - To add a description of the release:
-         - click edit
-         - in the 'describe this release' section add the content from the file `src/ontology/reports/mondo_release_diff.md`.
+    1. Add the release description to the release tab:
+          - All of the releases can be found under the [releases](https://github.com/monarch-initiative/mondo/releases) tab.
+          - To add a description of the release:
+                - click edit
+                - in the 'describe this release' section add the content from the file `src/ontology/reports/mondo_release_diff.md`.
```

Comment:
I don't see this file `src/ontology/reports/mondo_release_diff.md`.

Reply:
It is in gitignore you will only see it if you are the one running the release!

Reply:
Gotcha! That would be good to note in the docs.

Reply:
updated